starknet_py/tests/e2e/client/client_test.py

```diff
@@ -658,5 +694,5 @@ async def test_get_new_state_update(
 
     assert (
         ReplacedClass(contract_address=contract_address, class_hash=class_hash)
-        in state_update.state_diff.replaced_classes
+        in state_update.state_diff.replaced_classes  # pyright: ignore
```

Comment:
what's the error here? is it okay to silence it?

Reply:
The error is here because `get_state_update` now has return type `Union[BlockStateUpdate, StateUpdateWithBlock]`. The latter one doesn't have a field `state_diff`, because it is in `state_update.state_diff`. However, this test doesn't call the method with `include_block=True` argument, hence the ignore is okay, at least in my opinion.